From a520181562c2ab6514aa12f759bad9712f20d45c Mon Sep 17 00:00:00 2001
From: andrew <andrew@whartonlabs.com>
Date: Fri, 3 Feb 2017 11:47:52 +1100
Subject: [PATCH] Change argument check from String to UUID

The package should always be specified as its id (UUID) when calling the CloudApi. The normalisation of name/shortId/id to id is handled in TritonApi.
---
 lib/cloudapi2.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/cloudapi2.js b/lib/cloudapi2.js
index 90ebecf..f887714 100644
--- a/lib/cloudapi2.js
+++ b/lib/cloudapi2.js
@@ -782,7 +782,7 @@ CloudApi.prototype.getMachine = function getMachine(opts, cb) {
  */
 CloudApi.prototype.resizeMachine = function resizeMachine(opts, callback) {
     assert.uuid(opts.id, 'opts.id');
-    assert.string(opts.package, 'opts.package');
+    assert.uuid(opts.package, 'opts.package');
     var data = {
         action: 'resize',
         package: opts.package
-- 
2.21.0

